src/plone/exportimport/exporters/__init__.py

```diff
@@ -69,7 +69,10 @@ def export_site(self, path: Optional[Path] = None) -> List[Path]:
         with hooks.site(self.site):
             for exporter_name, exporter in self.exporters.items():
                 logger.debug(f"Exporting {self.site} with {exporter_name} to {path}")
-                paths.extend(exporter.export_data(path))
+                new_paths = exporter.export_data(path)
+                # could be None
+                if new_paths:
+                    paths.extend(new_paths)
```

Comment:
I would say an exporter should always return a list, even if it is empty. Then the code calling it doesn't have to check if it returned None.

Reply:
Ah, true, the type annotations say `export_data` should return `List[Path]`.
I have fixed the discussions exporter to return an empty list in this case.